analytics.html

```diff
@@ -0,0 +1,206 @@
+<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Crimson+Text" />
+<style>
```

Comment:
I brought this up in the meeting but all the css should be in a separate file. You can link to them in the <head></head> element.